scripts/v.unpack/v.unpack.py

```diff
@@ -77,7 +77,7 @@ def main():
     tar = tarfile.TarFile.open(name=input_base, mode="r")
     try:
         data_name = tar.getnames()[0]
-    except:
+    except Exception:
```

Comment:
```python
except (AttributeError, IndexError, tarfile.ReadError):
```

1. AttributeError: This exception is raised if the tar object does not have a getnames method. This could occur if tar is not a TarFile object.  
2. IndexError: This exception is raised during the tar.getnames()[0] call if the result of tar.getnames() does not have at least one element. This could occur if the tar file is empty.  
3. tarfile.ReadError: This exception is raised if the tar file cannot be read, which might happen if the file is not a tar file or is corrupted.